### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by qstorey**

### CODE REVIEW

**Comment by qstorey** [comment]
@hannesdejager are keen to review this PR?

File: src/main.rs [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+use ispot::logging;
+#[macro_use]
+extern crate log;
 
 fn main() {
     let matches = App::new("iTunes to Spotify")
         .version("0.1.0")
         .about("Convert an iTunes playlist to a Spotify playlist")
         .setting(AppSettings::SubcommandRequiredElseHelp)
+        .arg(Arg::with_name("debug")
```
Comment (hannesdejager) [inline src/main.rs]: You could possibly use a verbosity flag also i.e. `-v` means INFO, -vv means DEBUG -vvv means TRACE

```
        .arg(
            Arg::with_name(VERBOSITY)
                .short("v")
                .multiple(true)
                .help("verbosity level"),
        )
```

Comment (hannesdejager) [reply src/main.rs]: Do for log vs slog.

```
    let min_log_level = match arg_matches.occurrences_of(args::VERBOSITY) {
        0 => (slog::Level::Info, log::Level::Info),
        1 => (slog::Level::Debug, log::Level::Debug),
        2 | _ => (slog::Level::Trace, log::Level::Trace),
    };
```

Comment (qstorey) [reply src/main.rs]: Thanks. Switched to verbosity instead of debug/info level logging.

**Comment by hannesdejager** [comment]
LGTM

**Pull request merged**